src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -1,79 +1,53 @@
 package nextstep.qna.domain;
 
+import nextstep.qna.CannotDeleteException;
 import nextstep.qna.NotFoundException;
-import nextstep.qna.UnAuthorizedException;
 import nextstep.users.domain.NsUser;
 
-import java.time.LocalDateTime;
+public class Answer extends SoftDeletable {
+    private final BaseEntity contents;
```

Comment:
```suggestion
public class Answer extends SoftDeletableBaseEntity {
```
이름으로 봤을 때는 이와 같이 구현하는 것이 맞을지도..
객체 분리는 👍 
단, 이 객체를 어떻게 연결하는 것이 좋을지 고민해 보는 시간을 가질 것을 추천
즉, 객체의 의존은 상속과 조합이 있는데 어떻게 의존관계를 만들 것인지 고민해 보는 시간을 가지면 좋겠음.

Reply:
SoftDeletable로 해서 삭제 메서드 구현을 강제하고 싶은 마음에 요렇게 했었는데요..!
작성하면서도 상속과 조합을 같이 쓰는게 복잡도를 올린다(상속도 확인해보고 조합도 확인해봐야한다)는 생각이 들어 아쉬웠어서 말씀해주신 대로 조합만 사용하도록 변경했습니다.

SoftDeletableBaseEntity 로 생성했습니다!